hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/contracts/ContractsModule.java

```diff
@@ -261,8 +262,8 @@ static Predicate<Address> provideHederaPrecompiledContractsDetector() {
     static Predicate<Address> provideHederaSystemAccountDetector() {
         // all addresses between 0-750 (inclusive) are treated as system accounts
         // from the perspective of the EVM when executing Call, Balance, and SelfDestruct operations
-        return address ->
-                address.numberOfLeadingZeroBytes() >= 18 && Integer.compareUnsigned(address.getInt(16), 750) <= 0;
+        return address -> address.numberOfLeadingZeroBytes() >= 18
```

Comment:
Now that I look at it a second time that `address.numberOfLeadingZeroBytes() >= 18` is ... weird.  I know what it's doing, but ... couldn't the Besu guys have provided a method for it?

Well, we can't change `Address` or `DelegatingBytes` but maybe _we_ can have a properly-named method for this?  I see uses here and in `Version038AddressChecks`:  
![image](https://github.com/hashgraph/hedera-services/assets/117694041/e526f662-198b-4899-b7ee-f96d23f6c9d3)

And speaking of which, those constants 750 and 1000 appear in `Version038AddressChecks` with names.  It would be nice at some point in time to have a proper place to define these constants _once_.